### PR TITLE
Adding Word Search Classes

### PR DESCRIPTION
**Pull request by ynigoreyes**
Everything is tested and works. Thank you Tries!

Word List Tests (makes sure we can actually search for stuff and add words)
Screen Shot 2019-11-24 at 8 43 35 PM

WordFileProcessor (makes sure that we can convert a file filled with words into an array of strings)
Screen Shot 2019-11-24 at 10 03 14 PM

### CODE REVIEW

**Comment by ynigoreyes** [comment]
@Zeale hey man can you check out the code to see if you can use it?

File: src/oopgroop/projects/boggIe/api/WordFileProcessor.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package oopgroop.projects.boggIe.api;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class WordFileProcessor {
+    public static String[] ConvertFileToWordList(File wordFile) throws FileNotFoundException {
+        Scanner sc = new Scanner(wordFile);
```
Comment (Zeale) [inline src/oopgroop/projects/boggIe/api/WordFileProcessor.java:10]: This won't affect our project at all (probably), but typically you should close `Scanner`s after you're done using them.

Comment (Zeale) [reply src/oopgroop/projects/boggIe/api/WordFileProcessor.java:10]: I don't really know the difference between the "Add single comment" button and "Start a review" button, so I clicked add a single comment.

**Review by Zeale — APPROVED** [review]
Sorry this review is so late, I've had a hefty load of random garbage to get done before finals.

The only thing I wanted to say is that method names typically start with a lowercase letter. Like `doSomething()`, instead of `DoSomething()`. Idk if you're planning on using Java much later on, but in case, I thought I'd add that. :-)

**Comment by ynigoreyes** [comment]
Wait why did you delete the tests????? That was supposed to make sure nothing breaks if y’all wanted to change the code. @Zeale

**Pull request merged**